pghoard/transfer.py

```diff
@@ -39,6 +40,105 @@ class TransferOperation(StrEnum):
     Metadata = "metadata"
 
 
+class TransferIncrement(NamedTuple):
+    n_bytes: float
+    tracked_at: float
+
+
+class OperationEventProgressTracker(PGHoardThread):
+    """
+    Monitors the progress of operation events and updates metrics via clients (telegraph, prometheus, Pushgateway)
+    """
+
+    CHECK_FREQUENCY: int = 5  # check every 5 seconds for progress
+    WARNING_TIMEOUT: int = 5 * 60  # log a warning in case there is no progress during last 5 minutes
```

Comment:
**comment** Does these values sound reasonable?  I am not quite sure if they're a good threshold, will appreciate some suggestions